src/bridge/cockpitpackages.c

```diff
@@ -360,12 +361,31 @@ expand_libexec (const gchar *variable,
   return NULL;
 }
 
+static void
+apply_override (JsonObject *manifest,
+                const char *path)
+{
+  g_autoptr(GError) error = NULL;
+
+  g_autoptr(JsonObject) override = read_json_file (path, &error);
+  if (override)
+    {
+      cockpit_json_patch (manifest, override);
+    }
+  else
+    {
+      if (g_error_matches (error, G_FILE_ERROR, G_FILE_ERROR_NOENT))
+        g_debug ("no override found in %s", path);
+      else
+        g_message ("couldn't read %s: %s", path, error->message);
```

Comment:
I think that's a `g_warning()`, no?  Almost an `errx()` to be honest...

Reply:
Agreed. Changed to g_warning() in a separate commit, as that was existing behaviour. errx() is too strong, it's not an unrecoverable or logic error, just user misconfiguration (most probably, inaccessible files)